microsoft-edge/devtools-guide-chromium/css/index.md

```diff
@@ -22,7 +22,7 @@ ms.date: 11/16/2022
    limitations under the License.  -->
 # Get started viewing and changing CSS
 
-Follow the interactive tutorial sections in this article to learn the basics of using DevTools to view and change the CSS for a page.
+To learn the basics of using DevTools to view and change the CSS for a page, follow the interactive tutorial sections in this article.
 
 
 <!-- ====================================================================== -->
```

Comment:
change line 31 from:
Open the CSS Examples demo page in a new window or tab. To open the examples, right-click the link, or press and hold Ctrl (for Windows, Linux) or Command (for macOS) and then click the link.
to just:
Open the CSS Examples demo page in a new window or tab.

I find the added instructions confuse me instead of helping.  We can probably assume user can figure out how to accomplish opening a link in a new tab or window (ie by right-clicking the link and reading the list of menuitems).